src/Core/src/Platform/iOS/MauiScrollView.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using CoreGraphics;
+using UIKit;
+
+namespace Microsoft.Maui.Platform
+{
+	public class MauiScrollView : UIScrollView
+	{
+		public MauiScrollView()
+		{
+		}
+
+		// overriding this method it does not automatically scroll large UITextFields.
+		// Save the scrolling for KeyboardAutoManagerScroll.cs
+		public override void ScrollRectToVisible(CGRect rect, bool animated)
+		{
```

Comment:
Should this code check if the keyboard auto scroll code is enabled? 

If it's not then it will call to the base?

Reply:
Adding to this
Would it be possible to check with the KeyboardAutoScroll  if it's currently performing an operation? 

I worry about us just completely overriding this. 

Would it be possible to do something like

if (!KeyboardAutoScroll.IsCurrentlyDoingAScroll)
{
       base.ScrollRectToVisible(rect, animated) 
}

Reply:
I can try something like this!